docs/index.md

```diff
@@ -19,10 +19,9 @@ Axual Provider allows using Axual's Self-Service for Apache Kafka functionality
 	- To secure which applications are authorised to access streams, we support
 		- SSL (MUTUAL TLS) as a Certificate(PEM)
```

Comment:
Not related to this MR but: We should warn user in Terraform Docs about being able to create public environment but not being able to delete them.

Reply:
I'll add that.